packages/e2e-tests/src/specs/editor/saveStory.js

```diff
@@ -63,13 +63,6 @@ describe('Saving Story', () => {
     mockResponse = undefined;
   });
 
-  it('should display published dialog', async () => {
-    await createNewStory();
-    await insertStoryTitle('Test story');
-    await publishStory();
-    await expect(page).toMatch('Story published.');
```

Comment:
We already check for this inside of publish story.

```
await expect(page).toMatch('Story published.');
```

In which case this flow is already covered.